tensorboard/plugins/pr_curve/metadata.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Internal information about the pr_curves plugin."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from google.protobuf import json_format
+import tensorflow as tf
+from tensorboard.plugins.pr_curve import plugin_data_pb2
+
+
+PLUGIN_NAME = 'pr_curves'
+
+# Indices for obtaining precision and recall values from the tensor stored in a
```

Comment:
OK. If we are to use PEP 8, we should probably document that and do a repo-wide fix.